tf2_ros_py/tf2_ros/buffer.py

```diff
@@ -48,41 +60,49 @@ class Buffer(tf2.BufferCore, tf2_ros.BufferInterface):
 
     Stores known frames and offers a ROS service, "tf_frames", which responds to client requests
     with a response containing a :class:`tf2_msgs.FrameGraph` representing the relationship of
-    known frames. 
+    known frames.
     """
 
-    def __init__(self, cache_time = None, node = None):
+    def __init__(
+        self,
+        cache_time: Optional[Duration] = None,
+        node: Optional[Node] = None
+    ) -> None:
         """
         Constructor.
 
-        :param cache_time: (Optional) Duration object describing how long to retain past information in BufferCore.
-        :param node: (Optional) If node create a tf2_frames service, It responses all frames as a yaml
+        :param cache_time: Duration object describing how long to retain past information in BufferCore.
+        :param node: If node create a tf2_frames service, It responses all frames as a yaml
```

Comment:
I know you didn't change this, but since we are touching this line let's fix the wording:

```suggestion
        :param node: Create a tf2_frames service that returns all frames as a yaml document
```